### PR TITLE
Fixes #32805 - Overwrite outofsync_interval setting as host parameter

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
@ezr-ondrej Any changes required from my side?

**Review by ezr-ondrej — COMMENTED** [review]
Would it make sense to read also #{origin}_outofsync_interval in `reported_origin_interval`?

**Comment by nadjaheitmann** [comment]
IMHO, as both values are added anyways, it should be enough to make only one value overridable. If this changes, this still can be adjusted in a separate PR.  

https://github.com/theforeman/foreman/blob/develop/app/models/host_status/configuration_status.rb#L25

File: locale/foreman.pot [hunk 1]
```diff
@@ -8065,7 +8065,9 @@ msgid "Append domain names to the host"
 msgstr ""
 
 #: ../app/models/setting/general.rb:26
-msgid "Duration in minutes after servers are classed as out of sync."
+msgid ""
```
Comment (ezr-ondrej) [inline locale/foreman.pot:8068]: next time, this is done during the release process, so there is no need doing this manually in every PR, but doesn't hurt :)

**Review by ezr-ondrej — APPROVED** [review]
Nice addition, I hope it won't be much slower, but it would be slow only if someone missuses params and have too much of them.

**Comment by ezr-ondrej** [comment]
Thanks @nadjaheitmann ! :+1:

**Pull request merged**